widgets/hits.js

```diff
@@ -2,7 +2,12 @@ var React = require('react');
 
 var utils = require('../lib/utils.js');
 
-function hits({container = null, templates = {}, hitsPerPage = 20}) {
+function hits({
+    container = null,
+    templates = {},
+    transformData = {},
```

Comment:
If we go that way, why not `transform` and `transforms` ? The discussion was opened in the issue, but we never got a final answer on this.